nano/node/node.cpp

```diff
@@ -1554,19 +1555,19 @@ startup_time (std::chrono::steady_clock::now ())
 	});
 	if (NANO_VERSION_PATCH == 0)
 	{
-		BOOST_LOG (log) << "Node starting, version: " << NANO_MAJOR_MINOR_VERSION;
+		logger.try_log ("Node starting, version: ", NANO_MAJOR_MINOR_VERSION);
```

Comment:
Guess 2 Node starting, Work pool, "Genesis block not found", Node ID also should be logged. "Using bootstrap rep weight" likely too

Reply:
Yep seems I missed adding any to node.cpp. Thanks